metadata.json

```diff
@@ -34,7 +34,9 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "18.04"
+        "18.04",
```

Comment:
we should also drop 18.04 since it's EoL

Reply:
will do in another pr